### ISSUE TITLE
"at-rule-empty-line-before" should not be considered when nesting media queries

### ISSUE THREAD

**Issue by MoOx** [stylelint/stylelint#657]
With stylelint-config-standard, I have the following errors

```
Expected empty line before at-rule (at-rule-empty-line-before) 
```

For the following code

```css
.logo {
  @media (width >= 50em) {
    width: 40rem;
  }
}
```

Not sure it's relevant :)

**Comment by davidtheclark** [comment]
Just use one of the options: http://stylelint.io/?/src/rules/at-rule-empty-line-before/README.md

@jeddy3 Do you want to change standard?

**Comment by MoOx** [comment]
Sorry I should have opened this issue overthere first. Will do next time :)

**Comment by jeddy3** [comment]
FYI, released in [`2.0.0`](https://github.com/stylelint/stylelint-config-standard/releases/tag/2.0.0)

**Comment by MoOx** [comment]
I am using it since this morning. This issue has been opened after having upgraded.

**Comment by jeddy3** [comment]
Oops, my bad. I only changed the behaviour of nested comments, but not nested at-rules.

Fixed in `3.0.0`. 

Sorry about that...

**Comment by MoOx** [comment]
Np, thanks!
